### DISCUSSION TITLE
is and isNot Filter

### DISCUSSION THREAD

**Discussion by dotsinspace** [prisma/prisma#7220] [Q&A]
I am trying to get rating from database using following query.

`
await Context.DataBase.accountTruckpeRideTruckpeRatingReview.aggregate({
            'avg': {
              'TruckpeRatingReview': {
                'count': true,
                'max': true
              }
            },
            'where': {
              'TruckpeRide': {
                'TruckpeRideTruckpeDocument': {
                  'some': {
                    'TruckpeDocument': {
                      'TruckpeDriverTruckpeDocument': {
                        'TruckpeDriver': {
                          'Account': {
                            'id': accountId ?? Context.Session.user.id
                          }
                        }
                      }
                    }
                  }
                }
              }
            }
          })
`

But it always fails with following error.
`
Unknown arg `TruckpeRideTruckpeDocument` in where.TruckpeRide.TruckpeRideTruckpeDocument for type TruckpeRideRelationFilter. Available args:

```
type TruckpeRideRelationFilter {
  is?: TruckpeRideWhereInput | Null
  isNot?: TruckpeRideWhereInput | Null
}
```
`

NOTE: TruckpeRideTruckpeDocument is [] relationship to truckpeRide

Could you please tell me what is and isNot means in prisma or is there any doc reference.

**Comment by ryands17** [comment]
@dotsinspace 👋 

Could you share the relevant schema so that I can check?
